oci/oci.go

```diff
@@ -591,7 +592,22 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 
 	if c.state.Status == ContainerStateStopped {
 		exitFilePath := filepath.Join(r.containerExitsDir, c.id)
-		fi, err := os.Stat(exitFilePath)
+		var fi os.FileInfo
+		err = kwait.ExponentialBackoff(
+			kwait.Backoff{
+				Duration: 500 * time.Millisecond,
+				Factor:   1.2,
+				Steps:    6,
+			},
+			func() (bool, error) {
+				var err error
+				fi, err = os.Stat(exitFilePath)
+				if err != nil {
```

Comment:
Should we check if err is not ENOEXIST and return the error?  Should fi be _?

Reply:
Why node
if _, err :=  os.Stat(exitFilePath); err != nil {
                                        if os.IsNotExist(err) {
					        // wait longer
					        return false, nil
                                        } else {
                                                return true, err
                                       }
}

Reply:
I'm using `fi` below so there's no harm. As for not exists, I kept the old logig where we where checking ANY error. Your call BTW, I'm a bit reluctant to change logic fwiw

Reply:
Ok I missed the fi, The only advantage of checkinf the err is NOTEXIST, is that it will fail quicker. But I am fine with leaving it the way it is, if that is what you want.

Reply:
I agree with Dan on the fail fast but we can add that as a future PR.